syft/frameworks/torch/fl/utils.py

```diff
@@ -29,60 +29,67 @@ def extract_batches_per_worker(federated_train_loader: sy.FederatedDataLoader):
     return batches
 
 
-def add_model(dst_model, src_model):
+def add_model(dst_model, src_model,dst_no_data,src_no_data):
```

Comment:
`src_num_data` and `dst_num_data` might be clearer here; I thought these were booleans at first.

Reply:
Thank you for the advice, I changed here and also the tot_num_data